app/views/docs/authentication-management.phtml

```diff
@@ -0,0 +1,397 @@
+
+void main() { // Init SDK
+  Client client = Client();
+  Users users = Users(client);
+
+  client
+    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+    .setKey('919c2d18fb5d4...a2ae413da83346ad2') // Your secret API key
+  ;
+
+  Future result = users.updateLabels(
+    userId: '[USER_ID]',
+    labels: [ Role.label('subscriber') ],
+  );
+
+  result
+    .then((response) {
+      print(response);
+    }).catchError((error) {
+      print(error.response);
+  });
+}</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Kotlin</h3>
+        <div class="ide margin-top-small" data-lang="kotlin" data-lang-label="Kotlin SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Users
+import io.appwrite.Role
+
+suspend fun main() {
+    val client = Client(context)
+      .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+      .setProject("5df5acd0d48c2") // Your project ID
+      .setKey("919c2d18fb5d4...a2ae413da83346ad2") // Your secret API key
+
+    val users = Users(client)
+    val response = users.updateLabels(
+        userId = "[USER_ID]",
+        labels = [ Role.label('subscriber') ]
+    )
+    val json = response.body?.string()
+}</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Swift</h3>
+        <div class="ide margin-top-small" data-lang="swift" data-lang-label="Swift SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+func main() async throws {
+    let client = Client()
+      .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+      .setProject("5df5acd0d48c2") // Your project ID
+      .setKey("919c2d18fb5d4...a2ae413da83346ad2") // Your secret API key
+    let users = Users(client)
+    let response = try await users.updateLabels(
+        userId: "[USER_ID]",
+        labels: [ Role.label('subscriber') ]
+    )
+
+    print(String(describing: response)
+}</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>.NET</h3>
+        <div class="ide margin-top-small" data-lang="csharp" data-lang-label=".NET SDK">
+            <pre class="line-numbers"><code class="prism language-csharp" data-prism></code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>
+    This would correspond with the permissions <code>Permissions.read(Role.label('subscriber'))</code>, <code>Permissions.update(Role.label('subscriber'))</code>, <code>Permissions.delete(Role.label('subscriber'))</code>, and <code>Permissions.create(Role.label('subscriber'))</code>.
+</p>
+
+
+<p>
+    <a href="/docs/permissions" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Learn more about permissions</a>
+</p>
+
+<h2><a href="#teams" id="teams">Teams</a></h2>
```

Comment:
For this section, compared to others, we don't have code examples

Reply:
We didn't have time this time round. We can add in a future iteration.

Reply:
I know, this was squeezed in. We'll have a separate PR for this in depth.